pdf_build_src/cover.tex

```diff
@@ -0,0 +1,27 @@
+% adds the bids logo as the cover page of the pdf
+\begin{titlepage}
+
+\newcommand{\HRule}{\rule{\linewidth}{0.5mm}} % Defines a new command for the horizontal lines, change thickness here
+
+\center % Center everything on the page
+
+
+
+%----------------------------------------------------------------------------------------
+%	LOGO SECTION
+%----------------------------------------------------------------------------------------
+
+\includegraphics[width=0.6\textwidth]{images/BIDS_logo.jpg}\\[1cm] 
+
+%----------------------------------------------------------------------------------------
+%	TITLE SECTION
+%----------------------------------------------------------------------------------------
+
+\HRule \\[0.4cm]
+{ \huge \bfseries Brain Imaging Data Structure Specification}\\[0.4cm] % Title of your document
+\HRule \\[1.5cm]
+
+% \textsc{\large v1.2.1}\\[0.5cm]{\large 2019-08-14}\\[2cm]
+
+% \vfill % Fill the rest of the page with whitespace
+\textsc{\large v1.2.2}\\[0.5cm]{\large 2020-02-12}\\[2cm]\vfill\end{titlepage}
```

Comment:
EDIT: I realize now that you DO automatically extract version and date within your `pdf_build_src/process_markdowns.py` file.

However, please extract the version from the `mkdocs.yml`, because that's the file we mainly update for each release, as described in [the protocol](https://github.com/bids-standard/bids-specification/blob/master/Release_Protocol.md)

Also: I would highly recommend to "inject" the following line into the `cover.tex` file **during the build process**:

`\textsc{\large v1.2.2}\\[0.5cm]{\large 2020-02-12}\\[2cm]\vfill\end{titlepage}`

instead of having it already there and then merely update it during each build.

The advantage of this is, that for each release, `git` will not have to track changes of `cover.tex` ... and the functionality would be the same.

Also, it'd be less confusing to reviewers. 

In short: remove line `27` from `cover.tex` and instead *format* this line in `process_markdowns.py` and also write it to `cover.tex` FROM `process_markdowns.py` **during the build process**

Do similarly for `header.tex`

Reply:
Fixed this for both `header.tex` and `cover.tex`